### PR TITLE
Scripts for increasing versions

### PR DESCRIPTION
**Pull request by vegaro**
We've had a couple instances where we didn't increase the versions numbers correctly. I created a couple scripts for increasing purchases-hybrid-common version and to increase the version number of the plugin when preparing a new release.

### CODE REVIEW

File: scripts/prepare-for-release.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+NEW_VERSION=$1
+
+sed -i.bck "s/$PREVIOUS_VERSION/$NEW_VERSION/" package.json
+sed -i.bck "s/$PREVIOUS_VERSION/$NEW_VERSION/" ios/RNPurchases.m
+sed -i.bck "s/$PREVIOUS_VERSION/$NEW_VERSION/" android/build.gradle
+
+yarn example
+pushd examples/purchaseTester
+npx pod-install
+popd
```
Comment (aboedo) [inline scripts/prepare-for-release.sh:12]: should we just have them in fastlane, like we do for the other repos? I think the more consistent we can get with our automations, the better

Comment (vegaro) [reply scripts/prepare-for-release.sh:12]: makes sense. I figured that it was overkill for a couple of sed commands, but we should do it for when we start adding more automations

Comment (aboedo) [reply scripts/prepare-for-release.sh:12]: well, we did have scheduled for this sprint to fully automate deploys for hybrids

Comment (aboedo) [reply scripts/prepare-for-release.sh:12]: so it'll help as a foundation for that

Comment (vegaro) [reply scripts/prepare-for-release.sh:12]: agreed

**Comment by taquitos** [comment]
I migrated the scripts to a fastfile: https://github.com/RevenueCat/react-native-purchases/pull/269

**Comment by aboedo** [comment]
we should probably close this PR in favor of #269

**Pull request closed without merging**